automated_qc/tree_cover_loss_qc.py

```diff
@@ -0,0 +1,132 @@
+import os
+import math
+import numpy as np
+import rasterio as rio
+import fiona
+import pandas as pd
+import argparse
+from utils import parse_bounds, concatenate_windows
+from tqdm import tqdm
+from rasterio.windows import from_bounds
+from rasterstats import zonal_stats
+
+# parse arguments
+parser = argparse.ArgumentParser(description='Calculate tree cover loss for admin area')
+parser.add_argument('--admin_area', type=str, help='path to admin area shapefile')

```

Comment:
when you run QC for the data update, did you just plan to just run this script for each individual area? That might end up being a little time consuming for 10 areas * 4 analyses. Not sure the best way to make it easier though, since each area might have different contextual layers we need to check. Maybe a config file or something?